src/sidebar/store/create-store.js

```diff
@@ -26,63 +26,33 @@ const { createReducer, bindSelectors } = require('./util');
  * @param [any[]] middleware - List of additional Redux middlewares to use.
  */
 function createStore(modules, initArgs = [], middleware = []) {
-  // Create the initial state and state update function. The "base"
-  // namespace is reserved for non-namespaced modules which will eventually
-  // be converted over.
+  // Create the initial state and state update function.
 
   // Namespaced objects for initial states.
-  const initialState = {
-    base: null,
-  };
+  const initialState = {};
   // Namespaced reducers from each module.
-  const allReducers = {
-    base: null,
-  };
+  const allReducers = {};
   // Namespaced selectors from each module.
-  const allSelectors = {
-    base: {
-      selectors: {},
-      // Tells the bindSelector method to use local state for these selectors
-      // rather than the top level root state.
-      useLocalState: true,
-    },
-  };
-
-  // Temporary list of non-namespaced modules used for createReducer.
-  const baseModules = [];
+  const allSelectors = {};
 
   // Iterate over each module and prep each module's:
   //    1. state
   //    2. reducers
   //    3. selectors
   //
-  // Modules that have no namespace get dumped into the "base" namespace.
-  //
   modules.forEach(module => {
     if (module.namespace) {
       initialState[module.namespace] = module.init(...initArgs);
+
       allReducers[module.namespace] = createReducer(module.update);
       allSelectors[module.namespace] = {
         selectors: module.selectors,
       };
     } else {
-      // No namespace
-      allSelectors.base.selectors = {
-        // Aggregate the selectors into a single "base" map
-        ...allSelectors.base.selectors,
-        ...module.selectors,
-      };
-      initialState.base = {
-        ...initialState.base,
-        ...module.init(...initArgs),
-      };
-      baseModules.push(module);
+      console.warn('Module namespace value is missing', module);
```

Comment:
The term "module" is perhaps ambiguous if you see only the warning and are not looking at the code. "Store module does not specify a namespace" perhaps?